Source/Exporters/SVGKExporterUIImage.m

```diff
@@ -14,14 +14,19 @@ +(UIImage*) exportAsUIImage:(SVGKImage*) image antiAliased:(BOOL) shouldAntialia
 	if( [image hasSize] )
 	{
 		SVGKitLogVerbose(@"[%@] DEBUG: Generating a UIImage using the current root-object's viewport (may have been overridden by user code): {0,0,%2.3f,%2.3f}", [self class], image.size.width, image.size.height);
-		
-		UIGraphicsBeginImageContextWithOptions( image.size, FALSE, [UIScreen mainScreen].scale );
-		CGContextRef context = UIGraphicsGetCurrentContext();
+
+#if SVGKIT_MAC
+        CGFloat scale = [NSScreen mainScreen].backingScaleFactor;
+#else
+        CGFloat scale = [UIScreen mainScreen].scale;
+#endif
+		SVGGraphicsBeginImageContextWithOptions( image.size, FALSE, scale);
+		CGContextRef context = SVGGraphicsGetCurrentContext();
 		
 		[image renderToContext:context antiAliased:shouldAntialias curveFlatnessFactor:multiplyFlatness interpolationQuality:interpolationQuality flipYaxis:FALSE];
 		
-		UIImage* result = UIGraphicsGetImageFromCurrentImageContext();
-		UIGraphicsEndImageContext();
+		UIImage* result = SVGGraphicsGetImageFromCurrentImageContext();
+		SVGGraphicsEndImageContext();
```

Comment:
It's aim to keep macOS compatible. All the method call the `UIGraphics` method on iOS / tvOS. However, macOS does not have this convience method, you need to write long code to make it work (See the implementation).

Currently seems only one code use the method. So I can remove that from public API and put the implementation here as private API.

Reply:
Having looked at the utils class now, I think I'd prefer to see:

#if IPHONE
 UIGraphics....
#else
 SVGKGraphics...
#endif

NB:
1. the code path for iOS is clear and obvious
2. the caller still only has a simple case - one line for either OS
3. SVKG instead of SVG (SVG is reserved by the SVG spec)

Reply:
I'v remove the API from public into private. Renaming is OK.